extensions/wikia/CommunityPage/models/CommunityPageSpecialTopAdminsFormatter.class.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class CommunityPageSpecialTopAdminsFormatter {
+	const TOP_ADMINS_LIMIT = 10,
+		TOP_ADMINS_MODULE_LIMIT = 3;
+
+	/**
+	 * Returns array with fields to supply topAdmins.mustache template
+	 * If there are more than three admins provided returns first two and the count of remaining ones
+	 * @param array $topAdminsDetails
+	 * @return array
+	 */
+	public static function prepareData( array $topAdminsDetails ) {
+		$topAdminsCount = count( $topAdminsDetails );
+		$remainingAdminCount = self::prepareRemainingCount( $topAdminsCount );
+		return [
+			'admins' => self::prepareAdminsToShow( $topAdminsCount, $remainingAdminCount, $topAdminsDetails ),
+			'otherAdminCount' => $remainingAdminCount,
+			'haveOtherAdmins' => $remainingAdminCount > 0,
+			'adminCount' => $topAdminsCount,
+		];
+	}
+
+	private function prepareRemainingCount( $topAdminsCount ) {
+		return $topAdminsCount > self::TOP_ADMINS_MODULE_LIMIT
+			? $topAdminsCount - self::TOP_ADMINS_MODULE_LIMIT + 1
```

Comment:
@Grunny Do we have a convention on using ternary operators? I think we talked about them at some point.